apps/core/scenes/profile/Profile.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import * as React from 'react';
+import { ScrollView } from 'react-native';
+import { StyleSheet, Button, Card } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { margaritaTokens } from '@kiwicom/margarita-design-tokens';
+import { Illustration, Text } from '@kiwicom/margarita-components';
+
+type Props = {||};
+
+export default class Profile extends React.Component<Props> {
```

Comment:
Why, not just functional component?

Reply:
Because I'll need to handle sign-in buttons events inside class.

Reply:
OK so it is prepared for next PR, I understand. Thanks